sentry/src/main/java/io/sentry/instrumentation/file/FileInputStreamInitData.java

```diff
@@ -0,0 +1,27 @@
+package io.sentry.instrumentation.file;
+
+import io.sentry.IHub;
+import io.sentry.ISpan;
+import java.io.File;
+import java.io.FileInputStream;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+final class FileInputStreamInitData {
+
+  final @Nullable File file;
+  final @Nullable ISpan span;
+  final @NotNull FileInputStream delegate;
+  final @NotNull IHub hub;
```

Comment:
can we make them private?

Reply:
mm, and use getters instead? we can, but if they are final, is there a difference?

Reply:
not really, indeed